clients/java/src/main/java/io/camunda/client/impl/CamundaClientImpl.java

```diff
@@ -796,14 +797,21 @@ public CreateDocumentCommandStep1 newCreateDocumentCommand() {
 
   @Override
   public DocumentContentGetRequest newDocumentContentGetRequest(final String documentId) {
-    return new DocumentContentGetRequestImpl(httpClient, documentId, null, config);
+    return new DocumentContentGetRequestImpl(httpClient, documentId, null, null, config);
   }
 
   @Override
   public DocumentContentGetRequest newDocumentContentGetRequest(
       final DocumentReferenceResponse documentReference) {
+    final String contentHash =
+        (String)
+            documentReference.getMetadata().getCustomProperties().get(CONTENT_HASH_METADATA_KEY);
```

Comment:
I think we should include the document hash into the root-level properties of the document reference response along with the document ID and store ID, not in custom properties of the metadata.